### DISCUSSION TITLE
Problem with archive import

### DISCUSSION THREAD

**Discussion by AYOUR9** [aiidateam/aiida-core#6505] [Q&A]
Hey everyone,
-Here is the command used to create the archive: verdi archive create --groups "MY_GROUP_NAME" --input-calc-forward --input-work-forward MY_ARCHIVE_NAME.aiida
--And to import the groups, we use the following command: verdi archive import --group "MY_GROUP_NAME" --verbosity INFO MY_ARCHIVE_NAME
But in the group list, I did not find this group in my database. Here is the output I get when executing the command of import:

Processing 'create' Link(s)              100.0%|█████████████████████| 3705/3705
Processing 'return' Link(s)              100.0%|███████████████████████| 585/585
Processing 'input_calc' Link(s)          100.0%|█████████████████████| 7452/7452
Processing 'input_work' Link(s)          100.0%|█████████████████████| 6559/6559
Processing 'call_calc' Link(s)           100.0%|███████████████████████| 702/702
Processing 'call_work' Link(s)           100.0%|█████████████████████████| 16/16
Adding all Node(s) to the import Group     0.0%|                       | 0/12859
Collecting archive Node file keys        100.0%|███████████████████| 12859/12859

**Comment by agoscinski** [comment]
Hello @AYOUR9 the communication platform has been moved to discourse. Please ask your question there https://aiida.discourse.group
